### ISSUE TITLE
Cannot edit room names.

### ISSUE THREAD

**Issue by ThrashinVictim** [rand256/valetudo#476]
**Describe the bug**

**How to Reproduce**

**Expected behavior**

**Screenshots**

**Vacuum Model:** 
S5 Gen2

**Valetudo Version:** 
.10.7
**User-Agent** 

Apple Safari
<img width="586" alt="Screen Shot 2021-12-26 at 9 26 22 PM" src="https://user-images.githubusercontent.com/6667769/147428198-89145b8a-3c6c-4480-a45d-77cf4a83aa29.png">
 Version 15.2 (17612.3.6.1.6)

**Additional context**

**Comment by tylla** [comment]
It happened the same to me, same hardware, same Valetudo version, only I was using the webpage on Firefox (95.2.0) on my Android 9 phone. I spent lots of time trying to figure out a solution, until I gave up.
The next day it worked perfectly from Firefox (78.18.0esr) on my desktop machine and worked from the same phone as well. Must have been some intermittent error. Maybe give it another try. If it happens again and I manage to figure out some pattern I will report back.
